README.md

```diff
@@ -33,12 +33,6 @@ Run npm start in your local browser or using Live Server in Visual Studio Code.
 - GitHub: (https://github.com/rotshidzw)
 - Twitter: (https://twitter.com/footballlz )
 - LinkedIn: (https://www.linkedin.com/in/rochidzwa-chester-8062b6211/)
--
```

Comment:
- [ ] [OPTIONAL] Since it's a group project that would be great to add your partner info after yours ✌🏼. Good luck guys 🙌🏼🍀